.changeset/warm-lemons-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/backend': patch
```

Comment:
This is missing the `backend-function` package

Reply:
Updated!